src/pybind/mgr/volumes/fs/operations/trash.py

```diff
@@ -61,24 +61,30 @@ def purge(self, trashpath, should_cancel):
         """
         def rmtree(root_path):
             log.debug("rmtree {0}".format(root_path))
+
```

Comment:
unrelated line break.

Reply:
Can we keep the extra newlines characters I've added? It's easier to read code when block of code are separated not just by indentation but also by newlines.

Reply:
Let's do the in a new PR.